site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -442,19 +442,88 @@ After the certificates are generated, we will want to [inject caBundle](https://
 ```bash
 helm install my-release \
   --set agones.controller.disableSecret=true \
-  --set agones.controller.customCertSecretPath[0].key='ca.crt',customCertSecretPath[0].path='ca.crt'
-  --set agones.controller.customCertSecretPath[1].key='tls.crt',customCertSecretPath[1].path='server.crt'
-  --set agones.controller.customCertSecretPath[2].key='tls.key',customCertSecretPath[2].path='server.key'
-  --set agones.controller.allocationApiService.annotations={'cert-manager.io/inject-ca-from': 'agones-system/my-release-cert'} \
+  --set-string "agones.controller.customCertSecretPath[0].key=ca.crt" \
```

Comment:
I'm actually thinking - let's remove this top section with all the `--set` commands, I don't think anyone would do it in practice, and it's far easier to read with the yaml file.

So let's just keep the YAML file version (unless anyone disagrees?)

Reply:
Sure, changes added!